events/wc_primitive_events.txt

```diff
@@ -781,11 +785,153 @@ character_event = {
 	picture = GFX_evt_monk_water_deities
 	desc = EVTDESC_WCPRM_605
 	border = GFX_event_normal_frame_religion
-
+	
 	is_triggered_only = yes
-
+	
 	# Old fool!
 	option = {
 		name = EVTOPTA_WCPRM_605
 	}
 }
+
+character_event = {
+    id = WCPRM.500
+    picture = GFX_evt_kobold_statute
+    desc = EVTDESC_WCPRM_500
+    border = GFX_event_normal_frame_religion
+	
+    is_triggered_only = yes
+	
+	option = {
+	    name = EVTOPTA_WCPRM_500
+		piety = 1
+		wealth = -25
```

Comment:
So, you pay 25 for the decision and 25 for the option. Is it intended?

Reply:
Yes, cuz candle ritual is only for you and not included in for everyone, so if someone wants to do it he's gonna need to pay for it too